src/detect-engine-event.c

```diff
@@ -173,8 +173,13 @@ static DetectEngineEventData *DetectEngineEventParse (const char *rawstr)
     }
 
     if (OutdatedEvent(rawstr)) {
-        SCLogWarning(
-                SC_WARN_DEPRECATED, "decode-event keyword no longer supports event \"%s\"", rawstr);
+        if (SigMatchStrictEnabled(DETECT_DECODE_EVENT)) {
+            SCLogError(SC_WARN_DEPRECATED, "decode-event keyword no longer supports event \"%s\"",
+                    rawstr);
+            goto error;
+        } else {
+            SCLogNotice("decode-event keyword no longer supports event \"%s\"", rawstr);
```

Comment:
Also, is the Notice intentional?

Reply:
Yeah, wanted to reduce the severity as warning is known to be treated similar to error by some. In 7 it is still a warning.